### PR TITLE
Update team.yaml

### PR DESCRIPTION
**Pull request by xlxu220**
Add into the main content

### CODE REVIEW

File: book/team.yaml [hunk 1]
```diff
@@ -510,4 +510,26 @@ people:
...
+      supported by Canadian Space Agency.
+    expertise:
+    - Microwave Remote Sensing
+    - Signals of Opportunity (SoOP)
+    social:
+      - icon: github
+      icon_pack: fab
+      link: https://github.com/xlxu220
+    user_groups:
+      - SoOP Tutorial Lead
```
Comment (aaarendt) [inline book/team.yaml]: @xlxu220 this line is failing the spelling check. Can you remove "SoOP"? The user groups are unique names so keeping it consistent with the others is best. Thanks!

Comment (scottyhq) [reply book/team.yaml]: to skip spellcheck words or acronyms you can also add them to this list https://github.com/snowex-hackweek/website2022/blob/29fc416ec7a24d5d602b0773ed0706b02cf76afe/.github/workflows/qaqc.yaml#L36

File: book/team.yaml [hunk 2]
```diff
@@ -510,4 +510,26 @@ people:
...
-      - Tutorial Lead
\ No newline at end of file
+      - Tutorial Lead
+  # =============
+  - title: Xiaolan Xu	
+    avatar: https://avatars.githubusercontent.com/u/7988251?v=4
+    role: Tutorial Lead
+    organizations:
+    - name: Jet Propulsion Lab
+    - name: Radar Science
```
Comment (scottyhq) [inline book/team.yaml]:
```suggestion
    organizations:
      - name: Jet Propulsion Lab
      - name: Radar Science
```
make sure indents match levels for previous entries

File: book/team.yaml [hunk 3]
```diff
@@ -510,4 +510,26 @@ people:
...
+    bio: >
+      Expertise in the field of applied electromagnetics, electromagnetic wave propagation and scattering properties from snow-covered terrain, 
+      vegetated land surface, and bare soil, forward modeling, and retrieval algorithm development with applications to Earth remote sensing 
+      in space. She currently leads the NASA Earth Science Research project to improve snow water equivalence estimate using radar 
+      tomography technique and serves on the Science Team for NASA Soil Moisture Active and Passive Mission and Ku-band Radar mission 
+      supported by Canadian Space Agency.
+    expertise:
+    - Microwave Remote Sensing
+    - Signals of Opportunity (SoOP)
```
Comment (scottyhq) [inline book/team.yaml]:
```suggestion
    expertise:
      - Microwave Remote Sensing
      - Signals of Opportunity (SoOP)
```

File: book/team.yaml [hunk 4]
```diff
@@ -510,4 +510,26 @@ people:
...
+    social:
+      - icon: github
+      icon_pack: fab
+      link: https://github.com/xlxu220
```
Comment (scottyhq) [inline book/team.yaml]:
```suggestion
      - icon: github
         icon_pack: fab
         link: https://github.com/xlxu220
```

**Review by scottyhq — CHANGES_REQUESTED** [review]

File: book/team.yaml [hunk 5]
```diff
@@ -510,4 +510,26 @@ people:
...
+      in space. She currently leads the NASA Earth Science Research project to improve snow water equivalence estimate using radar 
+      tomography technique and serves on the Science Team for NASA Soil Moisture Active and Passive Mission and Ku-band Radar mission 
+      supported by Canadian Space Agency.
+    expertise:
+      - Microwave Remote Sensing
+      - Signals of Opportunity (SoOP)
+    social:
+      - icon: github
+         icon_pack: fab
+         link: https://github.com/xlxu220
```
Comment (scottyhq) [inline book/team.yaml]:
```suggestion
      - icon: github
        icon_pack: fab
        link: https://github.com/xlxu220
```

File: book/team.yaml [hunk 6]
```diff
@@ -512,7 +512,7 @@ people:
     user_groups:
       - Tutorial Lead
   # =============
-  - title: Xiaolan Xu	
+  - title: Xiaolan Xu
```
Comment (scottyhq) [inline book/team.yaml]: @jomey note this `-->` special character doesn't show up in the GitHub editor :(  would be nice to setup precommit to potential avoid some of the headaches around this yaml indent and formatting (https://github.com/uwhackweek/jupyterbook-template/issues/24)

Comment (jomey) [reply book/team.yaml]: As noted in the issue, an `.editorconfig` is perfect and designed for these kind of challenges.

Comment (scottyhq) [reply book/team.yaml]: @xlxu220 out of curiosity what text editor are you using to make changes locally?

**Review by scottyhq — APPROVED** [review]

File: book/team.yaml (hunk 6 above)
Comment (xlxu220) [reply book/team.yaml]: It's weird. I just use the mac TextEdit. I copy one of the previous people as a template and edit my own content to make sure of the format. However, it seems a lot of things changed.

**Pull request merged**